lib/pipeline.js

```diff
@@ -135,6 +135,82 @@ Pipeline.prototype = {
   },
 
   /**
+   * Bind to a streams events, and re-emit them on the pipeline
+   * @param {Stream} stream
+   * @param {String|Array} events
+   * @returns {Pipeline}
+   * @chainable
+   */
+  bind( stream, events ) {
+
+    var self = this
+
+    events = [].concat( events )
+
+    debug( 'bind', events )
+
+    events.forEach( function( event ) {
+
+      // Track bound events on the stream, to enable removal etc.
+      stream._pipageListeners = stream._pipageListeners || {}
+
+      // Prevent double-binding events
+      if( stream._pipageListeners[ event ] != null ) {
+        return
+      }
+
+      stream._pipageListeners[ event ] = function() {
+        var argv = Array.prototype.slice.call( arguments )
+        return self.emit.apply( self, [ event ].concat( argv ) )
```

Comment:
Note the `.apply()` – this is essentially `self.emit( event, ...argv )`, so no change in arguments from the usual event emission there.